modin/pandas/test/integration_test/__init__.py

```diff
@@ -0,0 +1,12 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
```

Comment:
Let's do a little renaming `modin/pandas/test/integration_test/__init__.py` -> `modin/pandas/test/integrations/__init__.py` like we have in `modin/examples/jupyter/integrations`.

Reply:
done